administrator/language/en-GB/com_content.ini

```
@@ -106,7 +106,7 @@ COM_CONTENT_FIELDSET_URLS_AND_IMAGES="Images and Links"
 COM_CONTENT_FILTER_AUTHORS_BY_ME="Created by me"
 COM_CONTENT_FILTER_FEATURED_NO="Unfeatured Articles"
 COM_CONTENT_FILTER_FEATURED_YES="Featured Articles"
-COM_CONTENT_FILTER_SEARCH_DESC="Search in title, alias and note. Prefix with ID: or AUTHOR: or CONTENT: to search for an article ID, article author or search in article content."
+COM_CONTENT_FILTER_SEARCH_DESC="Search in title, alias and note. Prefix with ID: or AUTHOR: or CONTENT: to search for an article ID, article author or search in article content. Prefix with CHECKEDOUT: to search for content checked out by an author."
```

```suggestion
COM_CONTENT_FILTER_SEARCH_DESC="Search in title, alias and note. Prefix with ID: or AUTHOR: or CONTENT: to search for an article ID, article author or search in article content. Prefix with CHECKEDOUT: to search for content checked out by a particular user."
```
The term "author" in the first sentence refers to the user who created the content (column `created_by`). 
In the 2nd sentence that term could be misleading. Before the last change it should have been "... checked out by the author" or "... checked out by its author". Now, after my last suggested code change has been applied, it can be any user who can edit content, not necessarily the author of the particular content or of any other content.

---

good spot thanks. I was misled by the issue report and didnt consider this.